### PR TITLE
Added support for ppc64le

### PR DESCRIPTION
**Pull request by junawaneshivani**
### What does this PR do?

I have added support for ppc64le architecture with reference to the [PR](https://github.com/eclipse/che-operator/pull/203) merged for s390x architecture.

### CODE REVIEW

**Comment by junawaneshivani** [comment]
@nickboldt PR now supports all architectures as suggested...

File: Dockerfile [hunk 1]
```diff
@@ -23,6 +23,7 @@ ADD . /go/src/github.com/eclipse/che-operator
 RUN case $(uname -m) in \
        x86_64) ARCH="amd64" ;; \
        s390x) ARCH="s390x";; \
+       ppc64le) ARCH="ppc64le";; \
```
Comment (nickboldt) [inline Dockerfile:26]: I might suggest making this less hard-coded and more generic... for s390x, ppc64le, and even future arches...

```
       x86_64) ARCH="amd64" ;; \
       *) ARCH="$1";; \
```

File: Dockerfile [hunk 2]
```diff
@@ -23,6 +23,7 @@ ADD . /go/src/github.com/eclipse/che-operator
...
 esac
 
 # do no break RUN lines when building with UBI base images. https://projects.engineering.redhat.com/browse/OSBS-7398 & OSBS-7399
```
Comment (nickboldt) [inline Dockerfile:29]: FYI this comment is no longer correct. Broken lines are allowed in Brew.

**Comment by nickboldt** [comment]
Followup https://github.com/eclipse/che-operator/pull/266

File: Dockerfile (hunk 1 above)
Comment (benoitf) [reply Dockerfile:26]: why merging instead of having the suggested proper fix for other arches ?

**Pull request merged**